code/datums/diseases/advance/symptoms/meme.dm

```diff
@@ -37,14 +37,14 @@
 	var/mob/living/carbon/M = A.affected_mob
 	if(prob(20 * A.stage) && !M.stat && !HAS_TRAIT(M, TRAIT_MINDSHIELD))
 		M.emote(emote)
-		if(A.stage >= 5)
-			for(var/mob/living/carbon/C in oviewers(M, 7))
+		if(A.stage >= 5 && prob(20))
+			for(var/mob/living/carbon/C in oviewers(M, 4))
 				var/obj/item/organ/eyes/eyes = C.getorganslot(ORGAN_SLOT_EYES)
 				if(!eyes || HAS_TRAIT(C, TRAIT_BLIND) || HAS_TRAIT(C, TRAIT_MINDSHIELD) || istype(C.get_item_by_slot(ITEM_SLOT_HEAD), /obj/item/clothing/head/foilhat))
 					continue
 				for(var/datum/disease/advance/D in C.diseases)
 					if(D.GetDiseaseID() == A.GetDiseaseID())
-						C.emote(emote)
+						continue
 				if(A.transmission >= 14)
```

Comment:
If its not this, then skip the loop entirely

Reply:
done